src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -162,9 +162,8 @@ private void addAliasToAllProjects(final OpenshiftRequestContext context, List<S
         if (context.isOperationsUser()) {
             // Check roles here, if user is a cluster-admin we should add
             LOGGER.debug("Adding indexes to alias '{}' for user '{}'", ADMIN_ALIAS_NAME, context.getUser());
-            buildAdminAlias(filteredProjects, projectPrefix);
-            filteredProjects.add(ADMIN_ALIAS_NAME);
```

Comment:
Why are we no longer adding the admin alias here if the user is an operations user? Where is that alias being added to the user at? -- I'm not seeing it in this file.